src/pages/personal-settings/index.js

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useState, useContext } from 'react'
+import { Grid, Box } from '@mui/material'
+import { useFormik } from 'formik'
+import toast from 'react-hot-toast'
+import WindowToolbar from 'src/components/Shared/WindowToolbar'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SelfServiceRepository } from 'src/repositories/SelfServiceRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { useResourceQuery } from 'src/hooks/resource'
+import { DataSets } from 'src/resources/DataSets'
+import * as yup from 'yup'
+import { Center } from 'devextreme-react/map'
+import { getStorageData } from 'src/storage/storageData'
+
+const PersonalSettings = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  const [initialValues, setInitialValues] = useState({
+    recordId: null,
+    email: '',
+    activeStatus: '',
+    userType: '',
+    homePage: '',
+    username: '',
+    fullName: '',
+    languageId: '',
+    menuTheme: '',
+    languageName: ''
+  })
+
+  useEffect(() => {
+    ;(async function () {
+      try {
+        const userData = getStorageData('userData')
+        const _userId = userData && userData.userId
+
+        const res = await getRequest({
+          extension: SelfServiceRepository.SSUserInfo.get,
+          parameters: `_recordId=${_userId}`
+        })
+
+        setInitialValues(res.record)
+      } catch (exception) {}
+    })()
+  }, [])
+
+  const { labels: _labels } = useResourceQuery({
```

Comment:
There's another function ig inside resource.js that fetches only labels and access. btw access is not being used here at all.